src/test/java/uk/gov/dwp/dataworks/integration/HealthCheckIntegrationTests.java

```diff
@@ -0,0 +1,224 @@
+package uk.gov.dwp.dataworks.integration;
```

Comment:
Would you say these are Integration tests or Unit? We have Mocks here, so is it a Unit test (of a module)?

Reply:
The rationale was that if I was hitting the service endpoints, i.e. the http endpoints exposed by the service with the mockmvc controller, then this is classed as an integration test as the entity under test is not an individual function but an entire facet of the service as a whole.  The unit test package has tests for the functions of the individual classes/providers.

Reply:
Sounds good to me :-)